src/config.js

```diff
@@ -42,4 +43,21 @@ export default {
         ['subscriber', process.env.ROLE_SUBSCRIBER],
         ['tester', process.env.ROLE_TESTER],
     ]),
+
+    messages: new Map([
+        ['activity', { id: process.env.MESSAGE_ACTIVITY, channel: 'iislander-board' }],
+        ['leaderboard', { id: process.env.MESSAGE_LEADERBOARD, channel: 'iislander-board' }],
+        [
+            'prime-achievements',
+            { id: process.env.MESSAGE_PRIME_ACHIEVEMENTS, channel: 'iislander-board' },
```

Comment:
These should maybe go to the factions file

Reply:
Any reason why? Things like faction channels aren't in the factions file. Also, this causes the editMessage function to not work. Plus then we would have 2 of these that work one way, and 3 that work another.